synapse/models/biz.py

```diff
@@ -0,0 +1,204 @@
+                    }),
+                    ('product', ('biz:product', {}), {
+                        'doc': 'The product included in the bundle.',
+                    }),
+                    ('deal', ('biz:deal', {}), {
+                        'doc': 'The deal which includes this bundle.',
+                    }),
+                    ('purchase', ('econ:purchase', {}), {
+                        'doc': 'The purchase which includes this bundle.',
+                    }),
+                )),
+                ('biz:product', {}, (
+                    ('name', ('str', {}), {
+                        'doc': 'The name of the product.',
+                    }),
+                    ('type', ('biz:prodtype', {}), {
+                        'doc': 'The type of product.',
+                        'disp': {'hint': 'taxonomy'},
+                    }),
+                    # TODO ('upc', ('biz:upc', {}), {}),
+                    ('summary', ('str', {}), {
+                        'doc': 'A brief summary of the product.',
+                        'disp': {'hint': 'text'},
+                    }),
+                    ('price:retail', ('econ:price', {}), {
+                        'doc': 'The MSRP price of the product.',
+                    }),
+                    ('price:bottom', ('econ:price', {}), {
+                        'doc': 'The minium offered or observed price of the product.',
+                    }),
+                    ('bundles', ('array', {'type': 'biz:bundle', 'uniq': True, 'sorted': True}), {
+                        'doc': 'An array of bundles included with the product.',
+                    }),
+                )),
+                ('biz:stake', {}, (
+                    ('vitals', ('ou:vitals', {}), {
+                        'doc': 'The ou:vitals snapshot this stake is part of.',
+                    }),
+                    ('org', ('ou:org', {}), {
+                        'doc': 'The resolved org.',
+                    }),
+                    ('orgname', ('ou:name', {}), {
+                        'doc': 'The org name as reported by the source of the vitals.',
+                    }),
+                    ('orgfqdn', ('inet:fqdn', {}), {
+                        'doc': 'The org FQDN as reported by the source of the vitals.',
+                    }),
+                    ('name', ('str', {}), {
+                        'doc': 'An arbitrary name for this stake. Can be non-contact like "pool".',
+                    }),
+                    ('asof', ('time', {}), {
+                        'doc': 'The time the stake is being measured. Likely as part of an ou:vitals.',
+                    }),
+                    ('shares', ('int', {}), {
+                        'doc': 'The number of shares represented by the stake.',
+                    }),
+                    ('invested', ('econ:price', {}), {
+                        'doc': 'The emount of money invested in the captable iteration.',
```

Comment:
Should captable be "cap table"? Just checking

Reply:
Good question, i'll check. :D